src/query/config/src/config.rs

```diff
@@ -1215,6 +1215,16 @@ pub struct QueryConfig {
     #[clap(long, default_value = "10000")]
     pub max_query_log_size: usize,
 
+    /// add session scope settings override
+    /// For example:
+    /// ```toml
+    /// [query.session_settings]
+    /// parquet_fast_read_bytes = "52428800"
+    /// max_storage_io_requests = "128"
+    /// ```
+    #[clap(skip)]
+    pub session_settings: HashMap<String, String>,
```

Comment:
I guess these fields can just put into the [query] or [storage] section instead of introducing a [session_settings] section.

The [session_settings] section may introduce some duplicated configuration entries with the [query] section or [storage] section.